MA-Bylaws.md

```diff
@@ -421,7 +423,11 @@ can vote. The same voting rules as for an Assembly meeting apply. Every
 member must have the possibility to inspect the submitted votes. A vote
 counter elected at the Assembly meeting summarizes the result and sends
 it to all members of MA. Election of persons is not possible by an
-electronic voting.
+electronic voting. However, elections of persons during a meeting can be 
+performed electronically in order that members present via teleconferencing 
+mechanisms can vote. For secret ballots, an appropriate anonymous service 
```

Comment:
These two sentences sound illogical and complicated to me and should be clarified. Either election of persons is not possible or it is. What about:

> Election of persons during a meeting can be performed electronically in order that members present via teleconferencing mechanisms can vote.

Reply:
It could be that it intends to differentiate between pure electronic voting and members present electronically at physical meetings. However, I agree with the general idea of your change, but we also have the next two sentences; and we then have two problems (even more if the consider the difference above):

- What happens if someone proposes "secret ballot" in an electronic voting and we haven't set it up?
- If we have a meeting at a conference where most people attend in person, should everyone still log in to the anonymous service?

One possibility would be something like:

Election of persons during a meeting can only be performed electronically if the agenda was announced together with an appropriate anonymous service and method for voting.  The anonymous service must be used for secret ballots.

Reply:
I agree with @beutlich that the current formulation is contradictory: we first stay that election of person is not possible by electronic voting, then in the next sentence we say that election of persons can be performed electronically. Is there any difference between electronic voting and elections performed electronically?

I have no strong opinions on making detailed rules as proposed by @HansOlsson, but we can't definitely keep the text as it is now.

Reply:
The current text is correct because **electronic voting** is precisely defined at the end of §14:

*An electronic voting has to be announced at least 2 weeks before the voting takes place, and the final proposal (e.g. consisting of a specification, or a library) must be generally available at least 1 week before the vote. The voting period is 6 working days. Every member of MA can vote. The same voting rules as for an Assembly meeting apply. Every member must have the possibility to inspect the submitted votes. A vote counter elected at the Assembly meeting summarizes the result and sends it to all members of MA. Election of persons is not possible by an electronic voting.*

This text cannot be changed, because we do not intend to allow election of persons during a 6 days vorting period, but only at the annual assembly meeting.

Therefore, the proposal from Thomas cannot be used.

I would not define all the precise details in the bylaws, because it is so much effort to change the bylaws. Therefore, I am against the proposal from Hans.

I would suggest to keep the proposal formulation